### PR TITLE
[13.0][MIG] account_invoice_validation_queued : Migration to 13.0

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
This is the migration of module : account_invoice_validation_queued

@Tecnativa TT22079

### CODE REVIEW

File: account_invoice_validation_queued/README.rst [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--invoicing-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-invoicing/tree/12.0/account_invoice_validation_queued
+    :alt: OCA/account-invoicing
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_validation_queued
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/95/12.0
```
Comment (Tardo) [inline account_invoice_validation_queued/README.rst]: Update readme

File: account_invoice_validation_queued/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#. Jobs are enqueued in the channel ``root.account_invoice_validation_queued``,
+   so you must adjust your
+   `Odoo configuration <https://github.com/OCA/queue/tree/11.0/queue_job#configuration>`_
```
Comment (Tardo) [inline account_invoice_validation_queued/readme/CONFIGURE.rst]:
```suggestion
   `Odoo configuration <https://github.com/OCA/queue/tree/13.0/queue_job#configuration>`_
```

File: account_invoice_validation_queued/wizards/validate_account_move.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        if invoices.filtered(lambda x: x.state not in "draft"):
+            # Let standard method to raise the exception about draft state
+            return self.post()
+        # Check if all invoices are for the same date
+        inv_date = invoices[:1].date
+        for invoice in invoices:
+            if invoice.date != inv_date:
+                raise exceptions.UserError(
+                    _("You can't enqueue invoices with different dates.")
+                )
```
Comment (Tardo) [inline account_invoice_validation_queued/wizards/validate_account_move.py]:
```suggestion
        if any(invoices.filtered(lambda x: x.date != inv_date)):
            raise exceptions.UserError(
                _("You can't enqueue invoices with different dates.")
            )
```

File: account_invoice_validation_queued/wizards/validate_account_move_view.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    <record id="validate_account_move_view" model="ir.ui.view">
+        <field name="model">validate.account.move</field>
+        <field name="inherit_id" ref="account.validate_account_move_view" />
+        <field name="arch" type="xml">
+            <button name="validate_move" position="before">
+                <button
+                    name="enqueue_invoice_confirm"
+                    string="Enqueue Validation"
+                    type="object"
+                    class="btn-primary"
```
Comment (Tardo) [inline account_invoice_validation_queued/wizards/validate_account_move_view.xml]:
```suggestion
                    class="btn btn-primary"
```

**Review by Tardo — CHANGES_REQUESTED** [review]
Looks good! Only a few comments...

File: account_invoice_validation_queued/README.rst (hunk 1 above)
Comment (manuelcalerosolis) [reply account_invoice_validation_queued/README.rst]: Fixed!

File: account_invoice_validation_queued/readme/CONFIGURE.rst (hunk 2 above)
Comment (manuelcalerosolis) [reply account_invoice_validation_queued/readme/CONFIGURE.rst]: Fixed!

File: account_invoice_validation_queued/wizards/validate_account_move.py (hunk 3 above)
Comment (manuelcalerosolis) [reply account_invoice_validation_queued/wizards/validate_account_move.py]: Great optimization!

File: account_invoice_validation_queued/wizards/validate_account_move_view.xml (hunk 4 above)
Comment (manuelcalerosolis) [reply account_invoice_validation_queued/wizards/validate_account_move_view.xml]: Fixed!

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@Tardo please re-check

**Comment by pedrobaeza** [comment]
Proceeding to merge as all is attended

/ocabot merge nobump

**Pull request merged**